static/javascript/tba_js/eventwizard_apiwrite.js

```diff
@@ -198,7 +198,7 @@ $('#results_file').change(function(){
             var match = matches[i];
 
             // check for invalid match
-            if(!match['Time']){
+            if(!match['Time'] || !match['Red 1']){
```

Comment:
Actually, time shouldn't be a mandatory field here. Just check for 'Red Score' and 'Blue Score'